pkg/identity/identity.go

```diff
@@ -22,11 +25,15 @@ type Identity interface {
 type Signer interface {
 	Sign(message []byte) ([]byte, error)
 }
+type Serializer interface {
```

Comment:
could you please check if we have chance to refactor with `internal/pkg/identity/identity.go`

Reply:
> could you please check if we have chance to refactor with `internal/pkg/identity/identity.go`

ok, merge into Signer interface.

```go
type Signer interface {
	Sign(message []byte) ([]byte, error)
	Serialize() ([]byte, error)
}
```